app/shared/contactUsTypes.ts

```diff
@@ -0,0 +1,34 @@
+interface BaseTopic {
+  id: string;
+  name: string;
+  selfServiceBox?: SelfServiceBox;
+  editableSubjectLine?: boolean;
+  noForm?: boolean;
+}
+
+interface SelfServiceBox {
+  text: string[];
+  linkText: string;
+  href: string;
+}
+
+export interface SubTopic extends BaseTopic {
+  subsubtopics?: BaseTopic[];
+  subsubTopicsTitle?: string;
```

Comment:
Why is the title not part of the `BaseTopic`?

Reply:
This variables stores the copy to be shown on the title of the box surrounding the subtopic and subsubtopic options. The topic does not have such a box and so it's not present in BaseTopic.